proto/initia/crypto/v1/derivable/keys.proto

```
@@ -0,0 +1,21 @@
+syntax = "proto3";
+package initia.crypto.v1.derivable;
+
+import "amino/amino.proto";
```

Buf (1.54.0)

import "amino/amino.proto": file does not exist

(COMPILE)